lib/typescript/httpclient/BUILD

```diff
@@ -17,6 +17,7 @@ ts_web_library(
         "@npm//@types/node",
         "@npm//camelcase-keys",
         "@npm//node-fetch",
+        "@npm//regenerator-runtime",
```

Comment:
@AudreyKj What do we need this for? 🤔 

Node environments should support `async` out of the box.